com.playonlinux.PlayOnLinux4.yaml

```diff
@@ -0,0 +1,148 @@
+
+cleanup:
+  - /share/pkgconfig
+  - /share/pixmaps
+  - /share/man
+  - /include
+
+inherit-extensions:
+  - org.freedesktop.Platform.Compat.i386
+  - org.freedesktop.Platform.ffmpeg_full.i386
+  - org.freedesktop.Platform.ffmpeg-full
+  - org.freedesktop.Platform.GL32
+  - org.freedesktop.Platform.VAAPI.Intel.i386
+  - org.winehq.Wine.DLLs
+  - org.winehq.Wine.gecko
+  - org.winehq.Wine.mono
+
+modules:
+  - python3-modules.yaml
+
+  - name: netcat
+    buildsystem: autotools
+    post-install:
+      - rm -rf $FLATPAK_DEST/share/locale/it
+      - rm -rf $FLATPAK_DEST/share/locale/sk
+    sources:
+      - type: archive
+        url: http://sourceforge.net/projects/netcat/files/netcat/0.7.1/netcat-0.7.1.tar.gz
+        sha256: 30719c9a4ffbcf15676b8f528233ccc54ee6cba96cb4590975f5fd60c68a066f
+        x-checker-data:
+            type: anitya
+            project-id: 21535
+            url-template: http://sourceforge.net/projects/netcat/files/netcat/$version/netcat-$version.tar.gz
+    cleanup:
+      - /man
+      - /info
+
+  - name: ImageMagick
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://imagemagick.org/archive/releases/ImageMagick-7.1.1-14.tar.gz
+        sha256: 2f52b6c9e4e0dfd8789e409b110dba2e806956ca96fb1b3c367887ff40ffb2d8
+        x-checker-data:
+            type: anitya
+            project-id: 1372
+            url-template: https://imagemagick.org/archive/releases/ImageMagick-$version.tar.gz
+
+  - name: icoutils
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: http://download.savannah.gnu.org/releases/icoutils/icoutils-0.32.3.tar.bz2
+        sha256: 17abe02d043a253b68b47e3af69c9fc755b895db68fdc8811786125df564c6e0
+        x-checker-data:
+            type: anitya
+            project-id: 1360
+            url-template: http://download.savannah.gnu.org/releases/icoutils/icoutils-$version.tar.bz2
+
+  - name: oniguruma
+    buildsystem: autotools
+    sources:
+      - type: archive
+        archive-type: tar-gzip
+        url: https://api.github.com/repos/kkos/oniguruma/tarball/v6.9.8
+        sha256: 1890b8c3150b65a6d761a72e3212a98d6b9c119a469c1c93099234bd3d6c5a0e
+        x-checker-data:
+            type: json
+            url: https://api.github.com/repos/kkos/oniguruma/releases/latest
+            version-query: .tag_name
+            url-query: .tarball_url
+
+  - name: jq
+    buildsystem: autotools
+    sources:
+      - type: archive
+        archive-type: tar-gzip
+        url: https://api.github.com/repos/jqlang/jq/tarball/jq-1.6
+        sha256: 30eaa22720cd899fd6b9b791bb079b0ae2745f9e3db59fd6730c807300c98156
+        x-checker-data:
+            type: json
+            url: https://api.github.com/repos/jqlang/jq/releases/latest
+            version-query: .tag_name
+            url-query: .tarball_url
+      - type: script
+        dest-filename: autogen.sh
+        commands:
+          - autoreconf -fi
+
+  - xterm.yaml
+
+  - scrot.yaml
```

Comment:
These 2 are Optional dependencies with submodules, I I keep them in extra files to keep the main Manifest clean and Maintenance easier.

Reply:
No it doesn't make maintenance easier.

There is no "optional' dependency with flatpak.

Reply:
> No it doesn't make maintenance easier.

I maintain some Flatpaks and for me it's easier. If you don't think that you can do it the way you like in your manifests.

> There is no "optional' dependency with flatpak.

IT#s a Optional dependency of PlayOnLinux, so it can be commented out for faster builds.